### ISSUE TITLE
Optimize for greyscale

### ISSUE THREAD

**Issue by olillin** [olillin/WebCal-Adapter#61]
The website should be just as useable in greyscale as with colour. This is for both color blind users but also users who use greyscale to reduce phone-addiction or any other reason.

The current biggest offender is the calendar picker selection. The faint glow and slight change in brightness is not enough. The partial checkboxes is also not great so #56 would also help with this
